### DISCUSSION TITLE
Documentation or guideline for runtime theming

### DISCUSSION THREAD

**Discussion by hiimnhan** [cloudscape-design/components#1121] [Q&A] [answered]
Hi Cloudscape,

I'm incredibly excited about the new feature `runtime-theming` you've introduced, but I must admit, I'm not entirely sure how to make the most of it. It seems like such a valuable addition, and I don't want to miss out on its potential benefits. It's really fit what I want to do now. Could you please provide me with a guideline or some instructions on how to utilize this feature effectively since the documentation is quite unclear?

**Comment by Al-Dani** [comment, marked as answer]
Hi @hiimnhan 

Basic idea is the following: you can provide custom values to a specific set of [tokens](https://cloudscape.design/foundation/visual-foundation/design-tokens/#tokens) to override default cloudscape styles.
You can apply new theme in runtime or build our components with new styles.

[Theming](https://cloudscape.design/foundation/visual-foundation/theming/) page has more details as well as some code snippets
To play around with different token values you can go to one of the [demos](https://cloudscape.design/demos/), click on Theme button in the upper right corner and add your values.
You can also check [this small codesandbox example](https://codesandbox.io/s/runtime-theme-cloudscape-xxdp9h-xxdp9h?file=/src/PageContent.tsx) with runtime theming.

Let me know if that helped.

Thank you!

**Reply by hiimnhan** [reply]
That's awesome! Thank you for your answer.
